data/kramdown-rfc2629.erb

```diff
@@ -46,13 +47,19 @@
 <%  venue = ps[:venue] -%>
 <%  if venue -%>
 <%    venue = KramdownRFC::ParameterSet.new(venue) -%>
-    <note title="Discussion Venues" removeInRFC="true">
+    <note title="About This Document" removeInRFC="true">
+      <t>
+<%   if latest = venue[:latest] -%>
+        The editor's copy of this draft can be found at <eref target="<%=latest%>"/>.
```

Comment:
(Sorry for nitpicking, but I'd like this to stabilize...)

This text implies a single editor.
Maybe "An editing draft of this document can be..."?

Reply:
```suggestion
        A copy of this draft with the latest changes can be found at <eref target="<%=latest%>"/>.
```

```suggestion
        The most recent version of this draft can be found at <eref target="<%=latest%>"/>.
```

WDYT?

Reply:
I have this slight allergy against the word "version"...
I'll trade that against "revision".

Reply:
(I'm really battling with the wording here as I think this needs to be useful for outsiders, who may know that an I-D has versions, and now are confronted with versions of these versions...)